recipe/meta.yaml

```diff
@@ -35,6 +35,9 @@ requirements:
     - nasm
     - sed
   host:
+    # GPU requirements
+    - cudnn                           # [cuda_compiler_version != "None"]
+    - nccl                            # [cuda_compiler_version != "None"]
```

Comment:
I think you forgot to add [cudatoolkit-dev](cudatoolkit-dev-feedstock)?

Reply:
I'm using https://github.com/conda-forge/pytorch-cpu-feedstock/blob/master/recipe/meta.yaml and https://github.com/AnacondaRecipes/tensorflow_recipes/blob/master/tensorflow-base-gpu/meta.yaml as templates, neither of which has cudatoolkit-dev. I also don't see any mention of it at https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cuda#cuda-builds

Reply:
Yeah `cudatoolkit-dev` isn't used for this `{{ compiler("cuda") }}` is (per the docs)

Reply:
You're right. I find the docs: https://conda-forge.org/docs/user/faq.html?highlight=cuda#faq-cuda-compiler-header